File: client/src/app/_dtos/admin/forwardedCategoryDto.ts
import { DetachedRouteHandle } from "@angular/router";
import { IForwardedOfficialDto } from "./forwardedOfficialDto";

export interface IForwardedCategoryDto
{
    id: number;
    orderItemId: number;
    categoryRefAndName: string;
    charges: number;
    forwardedOfficials: IForwardedOfficialDto[];
}

export class forwardedCategoryDto implements IForwardedCategoryDto
{
    id: number=0;
    orderItemId: number=0;
    categoryRefAndName: string='';
    charges: number=0;
    forwardedOfficials: IForwardedOfficialDto[]=[];
}

File: api/DTOs/Admin/Orders/ForwardedCategoryDto.cs
namespace api.DTOs.Admin.Orders
{
    public class ForwardedCategoryDto
    {
        public int Id {get; set;}
        public int OrderNo {get; set;}
        public DateTime OrderDate { get; set; }
		public int OrderItemId {get; set;}
		public string CategoryRefAndName {get; set;}
		public int Charges {get; set;}
		public ICollection<ForwardedOfficialDto> ForwardedOfficials {get; set;}
    }
}